### PR TITLE
feat: [FFM-2672]: Environments Listing Table UX standardisation

### PR DESCRIPTION
**Pull request by laura-mannus**
##### Summary:

-Fonts/heading/spacing are standard
-Tags icon/count from each row should now be removed
-Id tag style now  removed as Tags are not supported 

##### Jira Links:

(https://harness.atlassian.net/browse/FFM-2518)

##### Screenshots:

image

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by laura-mannus** [comment]
retrigger jest

File: src/modules/75-cf/pages/environments/EnvironmentsPage.tsx [hunk 1]
```diff
@@ -52,55 +49,27 @@ export const TypeCell = withEnvironment(({ environment }) => {
...
-    (acc: Array<{ name: string; value: string }>, [key, value]: [string, string]) => {
-      return [...acc, { name: key, value: value }]
-    },
-    [] as Array<{ name: string; value: string }>
-  )
   return (
     <Layout.Horizontal
       flex={{ distribution: 'space-between', align: 'center-center' }}
       padding={{ left: 'small', right: 'small' }}
       style={{ maxWidth: 'calc(100% - var(--spacing-medium))' }}
```
Comment (crystal-fong) [inline src/modules/75-cf/pages/environments/EnvironmentsPage.tsx:56]: extract this inline styling?

File: src/modules/75-cf/pages/environments/EnvironmentsPage.tsx [hunk 2]
```diff
@@ -52,55 +49,27 @@ export const TypeCell = withEnvironment(({ environment }) => {
...
     >
       <Layout.Vertical spacing="xsmall" style={{ maxWidth: 'calc(100% - 100px)' }}>
```
Comment (crystal-fong) [inline src/modules/75-cf/pages/environments/EnvironmentsPage.tsx:58]: here as well?

**Review by crystal-fong — DISMISSED** [review]
Just some tiny things, good work!

File: src/modules/75-cf/pages/environments/EnvironmentsPage.tsx (hunk 2 above)
Comment (laura-mannus) [reply src/modules/75-cf/pages/environments/EnvironmentsPage.tsx]: sure, done, thank you

File: src/modules/75-cf/pages/environments/EnvironmentsPage.tsx [hunk 3]
```diff
@@ -52,55 +50,27 @@ export const TypeCell = withEnvironment(({ environment }) => {
...
   return (
     <Layout.Horizontal
+      className={css.mediumMaxWidth}
       flex={{ distribution: 'space-between', align: 'center-center' }}
       padding={{ left: 'small', right: 'small' }}
-      style={{ maxWidth: 'calc(100% - var(--spacing-medium))' }}
     >
-      <Layout.Vertical spacing="xsmall" style={{ maxWidth: 'calc(100% - 100px)' }}>
-        <Text style={{ color: '#22222A', fontWeight: 500 }}>{environment.name}</Text>
+      <Layout.Vertical className={css.calcMaxWidth} spacing="xsmall">
```
Comment (chrisblakely01) [inline src/modules/75-cf/pages/environments/EnvironmentsPage.tsx:59]: wonder why do ya need this? I would have thought `Layout.Vertical` would stay within its container no?

File: src/modules/75-cf/pages/environments/EnvironmentsPage.tsx [hunk 4]
```diff
@@ -52,55 +50,27 @@ export const TypeCell = withEnvironment(({ environment }) => {
...
   const { getString } = useEnvStrings()
-  const tags = Object.entries(environment.tags ?? {}).reduce(
-    (acc: Array<{ name: string; value: string }>, [key, value]: [string, string]) => {
-      return [...acc, { name: key, value: value }]
-    },
-    [] as Array<{ name: string; value: string }>
-  )
   return (
     <Layout.Horizontal
+      className={css.mediumMaxWidth}
```
Comment (chrisblakely01) [inline src/modules/75-cf/pages/environments/EnvironmentsPage.tsx:55]: same as below, is this needed?

File: src/modules/75-cf/pages/environments/EnvironmentsPage.module.scss [hunk 5]
```diff
@@ -27,3 +27,11 @@
     margin-bottom: var(--spacing-medium);
   }
 }
+
+.mediumMaxWidth {
```
Comment (knagurski) [inline src/modules/75-cf/pages/environments/EnvironmentsPage.module.scss:31]: Try to name classes after what they apply to, not what they do.

File: src/modules/75-cf/pages/environments/EnvironmentsPage.tsx [hunk 6]
```diff
@@ -52,55 +50,27 @@ export const TypeCell = withEnvironment(({ environment }) => {
...
-      <Layout.Vertical spacing="xsmall" style={{ maxWidth: 'calc(100% - 100px)' }}>
-        <Text style={{ color: '#22222A', fontWeight: 500 }}>{environment.name}</Text>
+      <Layout.Vertical className={css.calcMaxWidth} spacing="xsmall">
+        <Text style={{ color: Color.BLACK, font: FontVariation.BODY2 }}>{environment.name}</Text>
 
         {environment.description && (
           <Container>
-            <Text style={{ color: '#6B6D85', fontSize: '12px', lineHeight: '18px' }}>{environment.description}</Text>
+            <Text style={{ font: FontVariation.SMALL, color: Color.GREY_600 }}>{environment.description}</Text>
           </Container>
```
Comment (knagurski) [inline src/modules/75-cf/pages/environments/EnvironmentsPage.tsx:65]: I'm pretty sure we could drop this `Container` as it isn't doing anything.

File: src/modules/75-cf/pages/environments/EnvironmentsPage.tsx [hunk 7]
```diff
@@ -52,55 +50,27 @@ export const TypeCell = withEnvironment(({ environment }) => {
...
         )}
 
         <Container padding={{ top: 'xsmall' }} margin={{ bottom: 'xsmall' }}>
-          <IdentifierText identifier={environment.identifier} inline style={{ padding: 'var(--spacing-xsmall)' }} />
+          <Text style={{ font: FontVariation.SMALL, color: Color.GREY_600 }}>
+            {getString('cf.environments.environmentID')} {environment.identifier}
+          </Text>
         </Container>
```
Comment (knagurski) [inline src/modules/75-cf/pages/environments/EnvironmentsPage.tsx:72]: Same here. The `Layout` is already adding the necessary spacing.

**Review by knagurski — COMMENTED** [review]
Some things for the future

File: src/modules/75-cf/pages/environments/EnvironmentsPage.module.scss [hunk 8]
```diff
@@ -27,3 +27,11 @@
...
+  max-width: 'calc(100% - var(--spacing-medium))';
+}
+
+.calcMaxWidth {
+  max-width: 'calc(100% - 100px)';
```
[the remark this answers is not in the document]
Comment (laura-mannus) [reply src/modules/75-cf/pages/environments/EnvironmentsPage.module.scss:36]: Updated in upcoming PR

File: src/modules/75-cf/pages/environments/EnvironmentsPage.tsx (hunk 4 above)
Comment (laura-mannus) [reply src/modules/75-cf/pages/environments/EnvironmentsPage.tsx:55]: Have removed these, don't know what was happening yesterday but now they seem to display fine without them :)

File: src/modules/75-cf/pages/environments/EnvironmentsPage.module.scss (hunk 5 above)
Comment (laura-mannus) [reply src/modules/75-cf/pages/environments/EnvironmentsPage.module.scss:31]: Have removed the styling as it doesn't do anything anymore

File: src/modules/75-cf/pages/environments/EnvironmentsPage.tsx (hunk 7 above)
Comment (laura-mannus) [reply src/modules/75-cf/pages/environments/EnvironmentsPage.tsx:72]: Removed in upcoming PR

**Pull request merged**